server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1255,9 +1265,54 @@ public void deleteConsumer(
                 returnCerts.add(cert);
             }
         }
+        // we want to insert the content access cert to this list
+        if (consumer.getContentAccessCert() != null) {
+            try {
+                returnCerts.add(contentAccessCertService.getCertificate(consumer));
+            }
+            catch (IOException ioe) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate."));
+            }
+            catch (GeneralSecurityException gse) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate."));
+            }
+        }
         return returnCerts;
     }
 
+    @ApiOperation(notes = "Retrieves the body of the Content Access Certificate for the Consumer",
+        value = "getContentAccessBody")
+    @ApiResponses({ @ApiResponse(code = 404, message = "") })
+    @GET
+    @Path("{consumer_uuid}/content")
+    @Produces(MediaType.APPLICATION_JSON)
+    public ContentAccessListing getContentAccessBody(
+        @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
+        @QueryParam("if-modified-since") Date since) {
+
+        log.debug("Getting content access certificate for consumer: {}", consumerUuid);
+        Consumer consumer = consumerCurator.verifyAndLookupConsumer(consumerUuid);
+        ContentAccessListing result = new ContentAccessListing();
+
+        // if there have been changes since the last cert generation
+        try {
+            ContentAccessCertificate cac = contentAccessCertService.getCertificate(consumer);
+
+            String cert = cac.getCert();
+            String json = cert.substring(cert.indexOf("-----BEGIN ENTITLEMENT DATA-----\n"));
+            json = json.substring(0, json.indexOf("-----END ENTITLEMENT DATA-----") + 30);
```

Comment:
`json` does not include signature yet... I left a [`FIXME`](https://github.com/candlepin/subscription-manager/blob/0790a62837a8e1c4c0944fb2e1e85e6bc8121303/src/subscription_manager/cache.py#L601) for the corresponding subscription manager change once this is fixed.